### PR TITLE
adding insights from study2 data

### PR DESCRIPTION
**Pull request by remo-help**
changelog:
- new lvo_template now has an entry for a location
- new file: lvo_template_no_passive.yml (self-explanatory)

- ReportItemStatus:
- - added new taxonomy class, ReportItemStatus with subdivision: ToolBroken
- - added a teamcommunication rule: report_broken_tool

- KnowledgeSharing:
- - added new rule in KnowledgeSharing: found_victim; this will trigger "I found a victim here"
- - added new rule for "I have a victim here"

- removed "find" from the search_triggers

- added "scout" as a trigger for search_specialist

- tests:
- - added tests for "found victim" and "switch role"

### CODE REVIEW

**Review by adarshp — APPROVED** [review]

File: src/main/resources/org/clulab/asist/grammars/team_communication.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ rules:
     label: RoleDeclare
     example: "I am an engineer"
     pattern: |
-      trigger = [lemma=/(?i)^be/]
+      trigger = [lemma=/(?i)^be|Im/ & tag=VBP]
```
Comment (BeckySharp) [inline src/main/resources/org/clulab/asist/grammars/team_communication.yml:68]: is this what the lemma form of I'm is?

File: src/main/resources/org/clulab/asist/grammars/team_communication.yml [hunk 2]
```diff
@@ -144,6 +151,33 @@ rules:
                             >/${positional_preps}/
       map: Map? = >/${positional_preps}/
 
+  - name: found_victim
+    priority: ${ rulepriority }
+    label: KnowledgeSharing
+    example: "I found a victim in the library"
+    pattern: |
+      trigger =[lemma=/(?i)^find|found/]
```
Comment (BeckySharp) [inline src/main/resources/org/clulab/asist/grammars/team_communication.yml:159]: maybe add `locate` and `spot` as a trigger

**Review by BeckySharp — COMMENTED** [review]
thanks! left some comments

File: src/main/resources/org/clulab/asist/grammars/team_communication.yml (hunk 2 above)
Comment (remo-help) [reply src/main/resources/org/clulab/asist/grammars/team_communication.yml]: done

**Pull request merged**